src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/config": "^5.0",
         "symfony/dependency-injection": "^5.3",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/event-dispatcher": "^5.1",
+        "symfony/event-dispatcher": "^5.3",
```

Comment:
should be reverted when `EventListenerAutoconfigurator` is removed

Reply:
Done.